foundation.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: teaspoons
+  description: |
+    Terra Scientific Pipelines Service, or teaspoons, facilitates running a number of defined scientific pipelines 
+    on behalf of users that users can't run themselves in Terra. The most common reason for this is that the pipeline
+    accesses proprietary data that users are not allowed to access directly, but that may be used as e.g. a reference panel
+    for imputation.
+  tags:
+    - java
+    - dsp
+    - terra
+    - springboot
+  annotations:
+    github.com/project-slug: databiosphere/terra-scientific-pipelines-service
+  links:
+    - url: https://grafana.dsp-devops.broadinstitute.org/d/3VhQ1_0Sz/teaspoons?orgId=1
+      title: Teaspoons Grafana Dashboard
+      icon: dashboard
+spec:
+  type: service
+  lifecycle: experimental
+  owner: teaspoons
```

Comment:
@choover-broad do you know how to create a new group in backstage? or should we just put devops here?